Docs/Authors.md

```diff
@@ -0,0 +1,8 @@
+# Authors
+
+Graphics Tools is a collaborative project containing contributions from individuals around the world. Our sincere thanks to all who have, and continue to contribute.
+
+- Alban Bergeret (AlbanBERGERET-Epic)
+- Cameron Micka (Cameron-Micka)
+- Daryl Obert
+- Luis Valverde (luis-valverde-ms)
```

Comment:
>Luis Valverde (luis-valverde-ms) [](start = 2, length = 32)

Thanks terribly nice. Not deserved!

Reply:
You've taken tons of time to review the releases, so I would say very deserved! This also made me realize I forgot @sw5813 which I've just added.